calibration/experiments/gcm_driven_scm/run_calibration.jl

```diff
@@ -79,23 +85,33 @@ end
 
 series_names = [ref_paths[i] for i in 1:length(ref_paths)]
 
-# minibatcher
+### define minibatcher
 rfs_minibatcher =
-    EKP.RandomFixedSizeMinibatcher(experiment_config["batch_size"])
+    EKP.FixedMinibatcher(collect(1:experiment_config["batch_size"]))
 observations = EKP.ObservationSeries(obs_vec, rfs_minibatcher, series_names)
 
-# ExperimentConfig is created from a YAML file within the experiment_dir
+###  EKI hyperparameters/settings
 @info "Initializing calibration" n_iterations ensemble_size output_dir
 CAL.initialize(
     ensemble_size,
     observations,
     prior,
     output_dir;
-    scheduler = EKP.DefaultScheduler(eki_timestep),
+    scheduler = EKP.DataMisfitController(on_terminate = "continue"),
```

Comment:
The PR message said "at default configuration" for the results displayed - but did it mean this?

Reply:
I mean default for this setup (ie. running the calibration with `experiment_config.yml`).